### ISSUE TITLE
Object St.Bin (0x55acbd186770), has been already deallocated[This problem makes the /var/log/syslog file super large, Dozens of GB]

### ISSUE THREAD

**Issue by ygcaicn** [micheleg/dash-to-dock#1360]
Ubuntu 20.04

```
Dec 11 13:42:33 hp-pavilion-laptop gnome-shell[3213]: Object St.Bin (0x55acbd186770), has been already deallocated — impossible to set any property on it. This might be caused by the object having been destroyed from C code using something such as destroy(), dispose(), or remove() vfuncs.
Dec 11 13:42:33 hp-pavilion-laptop gnome-shell[3213]: == Stack trace for context 0x55acb971f670 ==
Dec 11 13:42:33 hp-pavilion-laptop gnome-shell[3213]: #0   7ffe1714a050 I   /home/jachin/.local/share/gnome-shell/extensions/dash-to-dock@micxgx.gmail.com/docking.js:2036 (3758af4b9d30 @ 194)
Dec 11 13:42:33 hp-pavilion-laptop gnome-shell[3213]: == Stack trace for context 0x55acb971f670 ==
Dec 11 13:42:33 hp-pavilion-laptop gnome-shell[3213]: #0   7ffe1714a050 I   /home/jachin/.local/share/gnome-shell/extensions/dash-to-dock@micxgx.gmail.com/docking.js:2036 (3758af4b9d30 @ 194)
Dec 11 13:42:33 hp-pavilion-laptop gnome-shell[3213]: Object St.Bin (0x55acc2246a10), has been already deallocated — impossible to set any property on it. This might be caused by the object having been destroyed from C code using something such as destroy(), dispose(), or remove() vfuncs.
Dec 11 13:42:33 hp-pavilion-laptop gnome-shell[3213]: Object St.Bin (0x55acbd186770), has been already deallocated — impossible to set any property on it. This might be caused by the object having been destroyed from C code using something such as destroy(), dispose(), or remove() vfuncs.
Dec 11 13:42:33 hp-pavilion-laptop gnome-shell[3213]: == Stack trace for context 0x55acb971f670 ==
Dec 11 13:42:33 hp-pavilion-laptop gnome-shell[3213]: #0   7ffe1714a050 I   /home/jachin/.local/share/gnome-shell/extensions/dash-to-dock@micxgx.gmail.com/docking.js:2036 (3758af4b9d30 @ 194)
Dec 11 13:42:33 hp-pavilion-laptop gnome-shell[3213]: == Stack trace for context 0x55acb971f670 ==
Dec 11 13:42:33 hp-pavilion-laptop gnome-shell[3213]: #0   7ffe1714a050 I   /home/jachin/.local/share/gnome-shell/extensions/dash-to-dock@micxgx.gmail.com/docking.js:2036 (3758af4b9d30 @ 194)
Dec 11 13:42:33 hp-pavilion-laptop gnome-shell[3213]: Object St.Bin (0x55acc2246a10), has been already deallocated — impossible to set any property on it. This might be caused by the object having been destroyed from C code using something such as destroy(), dispose(), or remove() vfuncs.
Dec 11 13:42:33 hp-pavilion-laptop gnome-shell[3213]: Object St.Bin (0x55acbd186770), has been already deallocated — impossible to set any property on it. This might be caused by the object having been destroyed from C code using something such as destroy(), dispose(), or remove() vfuncs.
```

**Comment by repsorp** [comment]
Same issue

Ubuntu 20.10
GNOME Shell 3.38.1
Dash to dock v69

Restarting the extension stops error messages

**Comment by vanvugt** [comment]
This can probably be closed - it's a duplicate of #1332. And that's [Ubuntu bug 1905519](https://bugs.launchpad.net/bugs/1905519).
